packages/web-react/src/components/Dropdown/README.md

```diff
@@ -14,13 +16,15 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
 
 ## Props
 
-| Prop name          | Type                                         | Default    | Required | Description                                |
-| ------------------ | -------------------------------------------- | ---------- | -------- | ------------------------------------------ |
-| `id`               | `string`                                     | `<random>` | no       | Component id                               |
-| `renderTrigger`    | `(render: DropdownRenderProps) => ReactNode` | -          | no       | Properties for trigger render              |
-| `disableAutoClose` | `boolean`                                    | -          | no       | Disable close on click outside of Dropdown |
-| `UNSAFE_className` | `string`                                     | -          | no       | Wrapper custom classname                   |
-| `UNSAFE_style`     | `CSSProperties`                              | -          | no       | Wrapper custom style                       |
+| Prop name          | Type                                         | Default       | Required | Description                                  |
+| ------------------ | -------------------------------------------- | ------------- | -------- | -------------------------------------------- |
+| `id`               | `string`                                     | `<random>`    | no       | Component id                                 |
+| `renderTrigger`    | `(render: DropdownRenderProps) => ReactNode` | -             | no       | Properties for trigger render                |
+| `disableAutoClose` | `boolean`                                    | -             | no       | Disable close on click outside of Dropdown   |
```

Comment:
Could be `enableAutoClose` (or `isAutoCloseEnabled`, depending on what we use elsewhere) to better match the `data-autoclose` in `web` implementation.

Reply:
Renamed to `enableAutoClose` and set to breaking change.